samples/test-samples.cmake

```diff
@@ -71,6 +72,7 @@ foreach (SAMPLE data-sealing file-encryptor helloworld local_attestation remote_
     endif ()
 
     # Build with pkg-config
+    # TODO: Use INSTALL_DIR for pkg-config after it is made relocatable.
     execute_process(
       COMMAND ${CMAKE_COMMAND} -E env PATH=${INSTALL_DIR}/bin:$ENV{PATH} PKG_CONFIG_PATH=${INSTALL_DIR}/share/pkgconfig OE_PREFIX=${INSTALL_DIR} make -C ${SAMPLE_SOURCE_DIR} clean build simulate
```

Comment:
`PKG_CONFIG_PATH=${BUILD_DIR}/`

Reply:
Oof, we should de-duplicate if we can (later), this is like the third time I've made the correct change to one line and not the other. Thanks.